core/modules/system/system.menu.inc

```diff
@@ -109,6 +110,17 @@ function system_menu_block_form($config) {
     '#default_value' => $config['toggle'],
     '#description' => t('A menu toggle button, commonly known as a hamburger icon, allows the menu to be hidden initially on devices with small screens, but then easily expanded.'),
   );
+  $form['expand_all'] = array(
+    '#type' => 'checkbox',
+    '#title' => t('Expand all'),
```

Comment:
Can we change this text to `Show all menu links`. Checkbox labels should be self-explanatory whenever possible.